### PR TITLE
As an admin user I want to be able to delete topic

### PR DESCRIPTION
**Pull request by rtershak**

### CODE REVIEW

File: app/controllers/topics_controller.rb [hunk 1]
```diff
@@ -36,10 +36,18 @@ def update
...
 
+  def destroy
+    Topic.find(params[:id]).destroy
+    respond_to do |format|
+      format.html do
+        redirect_to topics_path, notice: 'Topic was successfully deleted.'
+      end
+    end
+  end
+
```
Comment (Kmalynss) [inline app/controllers/topics_controller.rb]: What will happen if there will be no topic with provided id.

File: app/controllers/topics_controller.rb [hunk 2]
```diff
@@ -36,10 +36,22 @@ def update
     end
   end
 
+  def destroy
+    if Topic.find(params[:id]).nil?
```
Comment (ruba-ruba) [inline app/controllers/topics_controller.rb]: you can omit nil

File: app/controllers/topics_controller.rb [hunk 3]
```diff
@@ -36,10 +36,22 @@ def update
...
+      redirect_to topics_path, notice: 'Topic does not exist.'
+    else
+      Topic.find(params[:id]).destroy
```
Comment (ruba-ruba) [inline app/controllers/topics_controller.rb]: use `Topic.find` only once

**Comment by serhiym** [comment]
closing - I assume that this is no longer needed

**Pull request closed without merging**